pages/index.js

```diff
@@ -14,6 +14,29 @@ import includes from 'lodash/includes';
 import Link from 'next/link';
 
 export default class extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      // default to changelog if no forumlink is found
+      releaseNote: {
+        link: '/changelog'
+      }
+    }
+  }
+
+  componentDidMount() {
+    fetch(`https://forums.resin.io/c/etcher.json`)
+    .then((res) => res.json())
+    .then((releaseNotes) => {
+      return releaseNotes.topic_list.topics.find(topic => ( topic.title.includes(locals.version)))
```

Comment:
I'm not sure if we want this automatic-detection - doesn't it mean that if a user posts a message to the forum like  "I'm having a problem with Etcher v1.1.0" then that'll be picked up as the release-notes topic?